### PR TITLE
Toastify validation

### PR DESCRIPTION
**Pull request by Dimdim28**
Added toastify feedbacks to places where errors can occure.

### CODE REVIEW

File: src/api/categoryAPI.ts
Comment (LikerFeed) [inline src/api/categoryAPI.ts:1]: good!

File: src/api/taskAPI.ts
Comment (LikerFeed) [inline src/api/taskAPI.ts:1]: nice

**Review by LikerFeed — APPROVED** [review]
Good work!

**Review by crazysparrow69 — APPROVED** [review]
ez

**Pull request merged**